build.assets/Dockerfile-centos7-fips

```diff
@@ -96,8 +119,6 @@ RUN yum groupinstall -y 'Development Tools' && \
     elfutils-libelf-devel-static \
     git \
     net-tools \
-    # required to create bindings for Rust's boring-rs crate
```

Comment:
Seems like we're including `/opt/llvm` via line 187 anyway, so this doesn't seem needed, as we overwrite it.